botocore/data/logs/2014-03-28/paginators-1.json

```diff
@@ -38,6 +38,12 @@
         "events",
         "searchedLogStreams"
       ]
+    },
+    "GetLogEvents": {
```

Comment:
You'll need to remove this, see: https://github.com/aws/aws-cli/blob/develop/tests/functional/logs/test_get_log_events.py

Might be a good idea to run through the CLI/boto3 tests to make sure there aren't other cases like this.